pkg/cache/v3/linear.go

```diff
@@ -144,6 +160,16 @@ func WithLogger(log log.Logger) LinearCacheOption {
 	}
 }
 
+// WithSotwStableVersions changes the versions returned in sotw to encode the list of resources known
+// in the subscription.
+// The use of stable versions for sotw also deduplicates updates to clients if the cache updates are
+// not changing the content of the resource.
+func WithSotwStableVersions() LinearCacheOption {
```

Comment:
nit, feel free to ignore since we already do that everywhere in this code, but: the go style is typically to preserve acronyms in a single case. So this should read: `WithSOTWStableVersions`. (tbh I was hesitant to even mention this, because I don't want to see the diff that changes it...)

Reply:
I can make the change on the last PR to avoid noise on this and the next one